include/global.berry.h

```diff
@@ -62,16 +75,21 @@ struct BattleEnigmaBerry
 
 struct BerryTree
 {
-    u8 berry;
-    u8 stage:7;
+    u8 berry:7;
+    u8 weeds:1;
+    u8 stage:3;
+    u8 mulch:4;
     u8 stopGrowth:1;
-    u16 minutesUntilNextStage;
-    u8 berryYield;
+    u16 minutesUntilNextStage:14;
+    u16 mutationA:2;
+    u8 berryYield:5;
+    u8 pests:1;
+    u8 mutationB:2;
     u8 regrowthCount:4;
-    u8 watered1:1;
-    u8 watered2:1;
-    u8 watered3:1;
-    u8 watered4:1;
+    u8 watered:4; // Used to keep track of bonuses in case of gradient watering
+    u16 moistureLevel:7;
```

Comment:
I haven't started testing this yet, but why doesn't this added word break saves? BerryTree is stored in SaveBlock1 afterall.

Reply:
Because the struct is padded by the compiler and this added word just recycles said padding.